module/VuFind/tests/unit-tests/src/VuFindTest/Log/Writer/Office365Test.php

```diff
@@ -30,8 +30,9 @@
 namespace VuFindTest\Log\Writer;
 
 use Laminas\Http\Client;
-use Laminas\Log\Formatter\Simple;
-use VuFind\Log\Writer\Office365;
+use Monolog\Level;
+use Monolog\LogRecord;
+use VuFind\Log\Handler\Office365Handler;
```

Comment:
Since the Log\Writer classes have been replaced with Log\Handler classes, we should move and re-namespace the tests to match (this one, for example, is still in the VuFindTest\Log\Writer namespace/file path, which is out of sync with the class being tested).